homeassistant/components/light/rfxtrx.py

```diff
@@ -22,12 +27,21 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Setup the RFXtrx platform. """
     lights = []
     devices = config.get('devices', None)
+
     if devices:
         for entity_id, entity_info in devices.items():
             if entity_id not in rfxtrx.RFX_DEVICES:
-                _LOGGER.info("Add %s rfxtrx.light", entity_info['name'])
-                rfxobject = rfxtrx.get_rfx_object(entity_info['packetid'])
-                new_light = RfxtrxLight(entity_info['name'], rfxobject, False)
+                _LOGGER.info("Add %s rfxtrx.light", entity_info[ATTR_NAME])
+
+                # Check if in push button mode
+                pushbutton = entity_info[ATTR_PUSHBUTTON] \
```

Comment:
Can we rename this to `fire_event`, it describes better what it will do.

Also, you can use the `get` method on a dict to specify a default value:
```python
fire_event = entity_info.get(ATTR_FIRE_EVENT, False)
```